google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolTest.java

```diff
@@ -272,6 +264,17 @@ private void setupMockSessionCreation() {
         .when(sessionClient)
         .asyncBatchCreateSessions(
             Mockito.anyInt(), Mockito.anyBoolean(), any(SessionConsumer.class));
+    if (options.getUseMultiplexedSession()) {
```

Comment:
here also: I think that you can remove the if statement, as it does not interfere with the non-multiplexed case